src/components/30-organisms/modal/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.1
```

Comment:
@axa-ch/modal has currently version 
2.0.4

Reply:
I can't see a 2.0.4 version somewhere, even in the pattern-library its 2.0.0
[https://patterns.axa.ch/?path=/story/components-modal--modal](url)

Reply:
You can see it on npm